src/mame/midway/midvunit.cpp

```diff
@@ -132,22 +132,35 @@ void midvplus_state::machine_reset()
 uint32_t midvunit_state::port0_r()
 {
 	uint16_t val = m_in0->read();
-	uint16_t diff = val ^ m_last_port0;
 
 	if (!machine().side_effects_disabled())
 	{
-		// make sure the shift controls are mutually exclusive
```

Comment:
Could keep this comment? It clarifies why the implementation is making an exception for the gearbox.